### PR TITLE
feat/defi-bridge > feat/ogv-merge

### PR DESCRIPTION
**Pull request by apexearth**

### CODE REVIEW

File: libs/defi/oeth/src/bridge/components/BridgeCard.tsx [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+  contracts,
+  getNativeToken,
+  getTokenId,
+} from '@origin/shared/contracts';
+import { ChainlinkCCIP } from '@origin/shared/icons';
+import {
+  ApprovalButton,
+  isNativeCurrency,
+  TxButton,
+  useTxButton,
```
Comment (toniocodo) [inline libs/defi/oeth/src/bridge/components/BridgeCard.tsx:27]: you'd need to use `useTxButton` from `@origin/defi/shared` to plug to the new notifications
Note that I added a `useApprovalButton` that works the same way to replace the `ApprovalButton` with a `TxButton`

File: libs/defi/shared/src/clients/wagmi.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ export const wagmiConfig = createConfig({
       ? fallback([
           http(
             `${import.meta.env.VITE_ALCHEMY_ARBITRUM_RPC}${
-              import.meta.env.VITE_ALCHEMY_ID_ARBITRUM
+              import.meta.env.VITE_ALCHEMY_ARBITRUM_ID
```
[the remark this answers is not in the document]
Comment (apexearth) [reply libs/defi/shared/src/clients/wagmi.ts:56]: In the OETH release it was given this name and it also matches the pattern above for the RPC. But we can hold off on changing this if it's too much of a headache.

File: apps/defi/src/components/Topnav/index.tsx
[the remark this answers is not in the document]
Comment (apexearth) [reply apps/defi/src/components/Topnav/index.tsx:1]: Maybe a merge fail

File: libs/defi/shared/src/clients/wagmi.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (apexearth) [reply libs/defi/shared/src/clients/wagmi.ts]: (I noticed it wasn't added to the env.d.ts file previously)

File: libs/defi/oeth/src/bridge/components/BridgeCard.tsx (hunk 1 above)
Comment (apexearth) [reply libs/defi/oeth/src/bridge/components/BridgeCard.tsx]: > Note that I added a useApprovalButton that works the same way to replace the ApprovalButton with a TxButton

Is that important to use now?

Comment (toniocodo) [reply libs/defi/oeth/src/bridge/components/BridgeCard.tsx]: yes, I changed the notifications, these are the right hooks for displaying them

File: libs/defi/shared/src/clients/wagmi.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (toniocodo) [reply libs/defi/shared/src/clients/wagmi.ts]: I mostly mean updating this in CI envs, in notion doc, and yes in the types

**Pull request merged**